### DISCUSSION TITLE
Add category for customManager

### DISCUSSION THREAD

**Discussion by marcm-ml** [renovatebot/renovate#27603] [Suggest an Idea]
It would be nice if we could add a category to a customManager (e.g. regex) and/or dependency groups. Currently, the category is empty (or n/a) for both.

I am currently adding labels to MRs based on the category. If it's a group MR or regex extracted dependency, the `categories` field is empty. My current workaround is that I use templating as follows: `"labels": ["renovate", "update::{{updateType}}", "category::{{#if categories}}{{{categories}}}{{else if isGroup}}group{{else}}other{{/if}}"],`. This works good enough, but it would be cool if in packageRules (or atleast in customManager), a category can be added.

**Comment by rarkins** [comment]
Sorry but it's not our intention that every manager needs a category, so I prefer not to force one for regex

**Reply by marcm-ml** [reply]
What I proposed would be a opt-in option, where if not given, it behaves the same as right now. What do you think?

**Reply by rarkins** [reply]
I prefer not to add this complexity if the primary reason is that one user can have a cleaner labels config

**Reply by marcm-ml** [reply]
Fair enough. Thanks for your quick reply!
